drivers/Aqara/aqara-lock/profiles/aqara-lock-battery.yml

```diff
@@ -0,0 +1,23 @@
+name: aqara-lock-battery
+components:
+- id: main
+  capabilities:
+  - id: lock
+    version: 1
+  - id: battery
+    version: 1
+  - id: tamperAlert
+    version: 1
+  - id: remoteControlStatus
+    version: 1
+  - id: firmwareUpdate
+    version: 1
+  - id: stse.lockCredentialInfo
+    version: 1
+  - id: stse.lockSystemInfo
```

Comment:
could you also include the definition for this capability?

Reply:
@seojune79 , `stse.lockSystemInfo` wouldn't used anymore, could you please remove this from profile?

Reply:
Yes. I removed it.